### ISSUE TITLE
gdal_viewshed example has output .png, but neglects showing the command line that created it

### ISSUE THREAD

**Issue by jidanni** [OSGeo/gdal#9433]
On https://gdal.org/programs/gdal_viewshed.html source code it says
```
Example
-------

Compute the visibility of an elevation raster data source with defaults


.. figure:: ../../images/gdal_viewshed.png
```
OK, that's great. But you will at least need a -xo and -yo, else you will get an error. And you will also need an input and output file too, etc.

So without the command line, the example leaves the user... in need of an example.

**Comment by jratike80** [comment]
I tried to understand the sample image, and obviously it is a combination of two viewshed anylysis. The yellow pixels (R=255 and G=255) show the area that if visible from the both observation locations (the green dots), while the small green area is only visible from one location. Interesting use case but without an explanation and commands rather hard to understand.

I can see that there was a line in the documentation that tried to explain this:
 `A computed visibility for two separate `-ox` and `-oy` points on a DEM.`
but it was removed by the commit https://github.com/OSGeo/gdal/commit/3d205f6cba5da0ac23093843db444db0d50167cb
that left the image orphaned.
